### DISCUSSION TITLE
A particular symmetry needs to be implemented

### DISCUSSION THREAD

**Discussion by Sreema1996** [QuSpin/QuSpin#638] [Q&A]
Dear developers,

I want to implement one symmetry in a spin Hamiltonian, which reads as \sum_{j}\sigma^z_{j}\sigma^z_{j+1} along with the total \sigma_{z} conservation. Can you please tell me how one can restrict the basis states employing this symmetry transformation?

**Comment by mgbukov** [comment]
your best shot is the [`user_basis`](
https://quspin.github.io/QuSpin/user_basis.html) -- check out the tutorial;
there are a few more sophisticated examples on the webpage too.

On Thu, Aug 3, 2023 at 7:57 AM Sreema1996 ***@***.***> wrote:

> Dear developers,
>
> I want to implement one symmetry in a spin Hamiltonian, which reads as
> \sum_{j}\sigma^z_{j}\sigma^z_{j+1} along with the total \sigma_{z}
> conservation. Can you please tell me how one can restrict the basis states
> employing this symmetry transformation?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/QuSpin/QuSpin/discussions/638>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADHSUBRFFDXFNBCSWPTNVJDXTM4V5ANCNFSM6AAAAAA3CHWRX4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Marin Bukov, PhD
Research Group Leader
Max Planck Institute for the Physics of Complex Systems
Noethnitzer Str. 38
Dresden 01187, Germany

url: https://www.pks.mpg.de/nqd
<http://quantum-dynamics.phys.uni-sofia.bg/>
